docs/source/conf.py

```diff
@@ -59,3 +63,39 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = []
+
+# `pydoctor_cwd` is the working directory used to call pydoctor.
+# `pydoctor_args` is the list of arguments used to call pydoctor when
+# generating the API docs via the Sphinx pydoctor extension.
+#
+# The following placeholders are available are resolved at runtime:
+# * `{outdir}` the Sphinx output dir
+#
+# Any output from pydoctor is converted into a Sphinx warning.
+#
+_git_reference = subprocess.getoutput('git rev-parse --abbrev-ref HEAD')
+if _git_reference == 'HEAD':
+    # It looks like the branch has no name.
+    # Fallback to commit ID.
+    _git_reference = subprocess.getoutput('git rev-parse HEAD')
+
+if os.environ.get('READTHEDOCS', '') == 'True':
+    rtd_version = os.environ.get('READTHEDOCS_VERSION', '')
+    if '.' in rtd_version:
+        # It looks like we have a tag build.
+        _git_reference = rtd_version
+
+
+pydoctor_cwd = str(pathlib.Path(__file__).parent.parent.parent)
```

Comment:
I'd prefer a name like `pydoctor_root` or `pydoctor_top`. The term `cwd` has the specific meaning of the current OS working dir, which is not what we're dealing with here. (Later I saw that we actually do make it the working dir, but it would still be better if that's an implementation decision that doesn't spill over into this configuration file.)

You don't need to convert to `str` here: in case we might want to use this directory for referencing other files later, it is more convenient to have it as a `Path` object.

Reply:
Well... this is no longer needed :)